store/postgres/secret_repository.go

```diff
@@ -226,9 +226,9 @@ func (repo *secretRepository) Delete(ctx context.Context, project models.Project
 
 	var result *gorm.DB
 	if namespace.Name == "" {
-		result = query.Where("namespace_id is null").Delete(&Secret{})
+		result = query.Where("namespace_id is null").Unscoped().Delete(&Secret{})
```

Comment:
Please add `Unscoped()` to the entire query.

```golang
query := repo.db.Unscoped().WithContext(ctx). <- Unscope at the query at line 223
```

Reply:
updated